### ISSUE TITLE
"9.3.1. A line of cards" doesn't form line

### ISSUE THREAD

**Issue by kdmsnr** [KanbanInAction/KanbanInActionErrata#4]
If you mean that forming a line of cards like product backlog, it makes sense. 

But this steps, just asking "Is this bigger or smaller than the first one?", cannot form a line of cards, because the card needs to be compared to a number of cards, not just  "the first one".

**Comment by marcusoftnet** [comment]
No, that's correct. It has to be compared to all the other ones and be inserted in the right spot. 

Point no 6 should be amended to handle that to improve clarity. Maybe something like 

```
Repeat steps 4 and 5 until you’re out of cards. Place each in the correct relative position, based on size, to the already laid out cards".
```

Great clarification. Thanks!
